common/config/tari_config_sample.toml

```diff
@@ -37,9 +37,18 @@
 # The number of liveness sessions to allow. Liveness sessions can be established by liveness monitors over TCP by
 # sending a 0x50 (P) as the first byte. Any messages sent must be followed by newline message no longer than
 # 50 characters. That message will be echoed back.
-# liveness_max_sessions = 0
-# liveness_allowlist_cidrs = ["127.0.0.1/32"]
-
+#liveness_max_sessions = 0
+#liveness_allowlist_cidrs = ["127.0.0.1/32"]
+
+# The buffer size for the base node publish/subscribe connector channel, connecting comms messages to the base
+# node (min value = 30).
+#buffer_size_base_node = 100
+# The buffer size for the base node wallet publish/subscribe connector channel, connecting comms messages to the
+# base node wallet (min value = 300).
+#buffer_size_base_node_wallet = 1000
+# The maximum amount of inbound messages to accept per second within any publish/subscribe connector channel - any
+# rate attemting to exceed this limit will be throttled (min value = 5).
+#buffer_rate_limit = 20
```

Comment:
Why was a default of 20m/s chosen? You're tests showed that 80m/s was optimal, correct?

Reply:
Oh I see, the buffer sizes were reduced.

Reply:
Values were, on a fast computer:
- 80 msg/s with buf_size of 50000 [no dropped messages]
- 50 msg/s with buf_size of 15000 [dropped messages].

_**Reasoning**: Going back to the original buf_size of 1000 means we have to be more conservative and limit the rate even more, so therefor I chose 20 msg/s._

Reply:
>I suspect a more fine-grained restock is preferable for performance: instead of 20m/s you would use 2 msgs per 100ms - otherwise I suspect the buffer will be under-utilised on average (trade off between memory usage and performance). In any case, I'm happy to try these values.

If the restock interval is 100ms, the lowest rate can be 10msg/s, whereas with a restock interval of 1000ms, the lowest rate can be 1msg/s. I chose 5msg/s as the lowest rate for any buffer, which still seems reasonable for normal use.